README.md

```diff
@@ -28,6 +28,14 @@ for lists:
 
     import RAKE
     Rake = RAKE.Rake(RAKE.SmartStopList());
+    
+#### Additional Parameters ####
+    import RAKE
+    Rake = RAKE.Rake([stopwords,
+                     [minimum_number_of_characters_each_word_should_have],
+                     [maximum_number_of_words_each_phrase_should_have],
+                     [minimum_number_of_times_each_keyword_should_appear]);
```

Comment:
yeah, this should just use the same variables as the extra function and include a little explanation for them.